src/db.rs

```diff
@@ -101,20 +109,31 @@ where
         let gc_interval = config.gc_interval;
         let gc_min_blocks = config.gc_min_blocks;
         let gc_target_duration = config.gc_target_duration;
+        let exit = Arc::new((Mutex::new(false), Condvar::new()));
```

Comment:
this tuple seems weird, but it is right from the parking lot condvar docs, so I guess that is how you do it...